### ISSUE TITLE
Missing baseline.json

### ISSUE THREAD

**Issue by PeterKottas** [relative-ci/bundle-stats#532]
My setup is fairly simple, webpack.config.js is located in the root of the project, I am also running the build script from the root. Yet the stats plugin is giving me an error about missing baseline.json. 
After taking a closer look, it appears that it attempts to find it at
```
..\node_modules\.cache\bundle-stats\baseline.json
```
In other words, outside of the project folder. Indeed by putting a node_modules/.cache/bundle-stats/baseline.json manually one folder up from the root, I can see that it loads succsesfully.
This happens on windows, bundle-stats version 1.15.2

**Comment by vio** [comment]
Hi @PeterKottas, thanks for reporting it! I assume you use the webpack plugin, is it possible to share the error message/stacktrace/node version/webpack version?

**Comment by PeterKottas** [comment]
No problem, here are the versions, there's no error, it simply says baseline missing. Yes, this is related to the webpack plugin.
```
webpack: 4.41.2
node: v10.13.0
```

**Comment by PeterKottas** [comment]
Just for completeness sake, this is input without baseline option:
```
Missing baseline stats, see "baseline" option.
```
And this with:
```
Missing baseline stats, see "baseline" option.
Write baseline data to ..\node_modules\.cache\bundle-stats\baseline.json
```
Then this chunk is emited
```
..\node_modules\.cache\bundle-stats\baseline.json
```
Weirdly, the path for the chunk is correct, as my output goes into build folder. I think this plugin is somewhere picking up wrong path from the context. 

This is my output section of webpack
```
output: {
      path: path.join(__dirname, './build'),
      filename: 'dist/js/[name].[hash].js',
      publicPath: '/', 
      chunkFilename: 'dist/js/chunk.[name].[hash].js',
      pathinfo: false,
    },
```

**Comment by vio** [comment]
Thanks for the extra info @PeterKottas! ;)

The behavior is as expected, though the message should be a [warning](https://github.com/relative-ci/bundle-stats/blob/master/packages/bundle-stats/src/webpack-plugin.js#L62), there might be an issue with the way webpack logger is being used.

By default, [compare mode](https://github.com/relative-ci/bundle-stats/tree/master/packages/bundle-stats#compare-mode) is enabled, so the plugin is expecting the file to be there if not will log a warning and the report will not show the baseline data. Once you will run a build with the `baseline` option enabled, the baseline data file will be saved and used as a comparison for the next builds.

**Comment by PeterKottas** [comment]
I think you misunderstood, it writes the baseline correctly, it, however, fails to read. Please re-read my initial (first) message.

**Comment by PeterKottas** [comment]
I agree about the warning, but that's really just cosmetics :) What I am seeing is an actual bug where it looks for the baseline outside of the project folder.

**Comment by PeterKottas** [comment]
To help you to debug further, it correctly writes the output to:
```
..\node_modules\.cache\bundle-stats\baseline.json
```
Because in the context of my "build" directory, that is correct.
However, when reading the baseline, it again tries to look at the same path (notice the 2 dots at the start of the string), but this time, it's incorrect. As in the context of the project folder, this goes outside. I hope this helps.

**Comment by vio** [comment]
That sounds like a bug, will have a look. Thanks for bearing with me! :)

**Comment by PeterKottas** [comment]
No worries, happy I could help! And thanks for looking into this.

**Comment by vio** [comment]
Fixed and published `1.15.3`, @PeterKottas can you please update the package and try again?

Thanks again for the extra details! ;)

**Comment by PeterKottas** [comment]
Worked like a charm, cheers!
